internal/support/notifications/rest_transmission.go

```diff
@@ -90,6 +90,52 @@ func transmissionBySlugHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func transmissionBySlugAndStartEndHandler(w http.ResponseWriter, r *http.Request) {
+
+	if r.Body != nil {
+		defer r.Body.Close()
+	}
+
+	vars := mux.Vars(r)
+	slug := vars["slug"]
+	start, err := strconv.ParseInt(vars["start"], 10, 64)
+	// Problem converting start
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		LoggingClient.Error("Error converting the start to an integer")
+		return
+	}
+	end, err := strconv.ParseInt(vars["end"], 10, 64)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		LoggingClient.Error("Error converting the end to an integer")
+		return
+	}
+	limitNum, err := strconv.Atoi(vars["limit"])
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
@weichou1229 These errors should all be `http.StatusBadRequest` since they would be thrown as the result of a type conversion failure based on incorrect URL values. I see, however, that this change is consistent with the rest of the logic in this controller -- so really all similar cases should return `http.StatusBadRequest`. Will changing this have a disruptive effect on tests?

Separately, please included `err.Error()` in the logged error message. Change error text to 
`fmt.Sprintf("failed to parse start %s %s", vars["start"], err.Error())`

Reply:
> Will changing this have a disruptive effect on tests?

No, there are no such tests. But you remind me, I need to add this test case.

Reply:
Changed, please help review.